llvm_util/cmd_args_def.h

```diff
@@ -50,6 +50,7 @@ if (!report_dir_created && !opt_report_dir.empty()) {
         newname = "in";
       newname += "_" + get_random_str(8) + ".txt";
       path.replace_filename(newname);
+      std::cerr << "\n\nWriting report to: " << newname <<  endl << endl;
```

Comment:
we can't do this. This will break the LLVM unit test suite. We can only add output when the test fails.

Reply:
> This will break the LLVM unit test suite. We can only add output when the test fails.

Writing to stderr rather than stdout was meant to avoid that.  The Lit tests I tried before submitting the PR all passed, but the whole Lit suite overnight on Linux got a failure rate of 0.38% (193/50546).  Some of the failures passed in isolation (e.g. pr25342.ll); one which didn’t (logical-right-shift-until-zero.ll) also failed without my change.  
    I may no longer need the report name in the log for cases in which it’s not output (presumably with `has_failure` false in tv.cpp:424), so should I revert that part of the PR?
    Redirecting verbose `opt-alive.sh` output to stderr still seems worth fixing (I warned about it in the ReadMe in 3b631cbe, and this should reduce the problem), and the Xcode hints may be helpful at least for some of us who left Apple.

Reply:
Yes, this bit needs to be reverted.
I know for a fact that it will break the LLVM test suite. That's why we only print the report file name on failure. There are tests that grep for stuff on stderr.